CreateReactApp: Module not found on build

Hey,


This
```
./node_modules/semantic-ui-less/semantic.less
ModuleNotFoundError: Module not found: Error: Can't resolve './definitions/elements/themes/default/assets/images/flags.png' in 'C:\Users\user_\portfolio-app\node_modules\semantic-ui-less'
```

has been happening to me in my create-react-app and I couldn't figure out what to do. I've started a new project, from scratch and this happened again so I believe I could either be missing a step, or there's a bug?


I've create a react app from scratch, installed semantic-ui-react, installed semantic-ui-less with craco and updated the package.json file. I've also included craco.config (should this be in /src?).
I've also installed semantic-ui-react/bootstrap package and updated the bottom of the theme.config file with the 4 lines, exactly as they are (should I be changing any values?)
I'm importing "import 'semantic-ui-less/semantic.less'" at the end of my index.js. 

Did I miss anything?

Thanks

--- a/docs/src/pages/Theming.mdx
+++ b/docs/src/pages/Theming.mdx
@@ -80,14 +80,14 @@ yarn remove semantic-ui semantic-ui-css
 
 ### Install required dependencies
 
-Create React App does not support LESS out of the box, we and propose to use [`@craco/craco`](https://www.npmjs.com/package/@craco/craco) to add it and avoid `eject`. You have to install required packages:
+Create React App does not support LESS out of the box, we and propose to use [`@craco/craco`](https://www.npmjs.com/package/@craco/craco) to add its support and avoid `eject`. You have to install required packages:
 
 ```bash
-npm install @craco/craco craco-less semantic-ui-less --save-dev
+npm install @craco/craco @semantic-ui-react/craco-less semantic-ui-less --save-dev
 ```
 
 ```bash
-yarn add @craco/craco craco-less semantic-ui-less --dev
+yarn add @craco/craco @semantic-ui-react/craco-less semantic-ui-less --dev
 ```
 
 And then update your `package.json` and create `craco.config.js`:
@@ -104,39 +104,8 @@ And then update your `package.json` and create `craco.config.js`:
 ```
 
 ```jsx label=craco.config.js
-const { getLoader, loaderByName, throwUnexpectedConfigError } = require('@craco/craco')
-
 module.exports = {
-  webpack: {
-    alias: {
-      '../../theme.config$': require('path').join(__dirname, '/src/semantic-ui/theme.config'),
-    },
-  },
-  plugins: [
-    { plugin: require('craco-less') },
-    {
-      plugin: {
-        overrideWebpackConfig: ({ context, webpackConfig }) => {
-          const { isFound, match: fileLoaderMatch } = getLoader(
-            webpackConfig,
-            loaderByName('file-loader'),
-          )
-
-          if (!isFound) {
-            throwUnexpectedConfigError({
-              message: `Can't find file-loader in the ${context.env} webpack config!`,
-            })
-          }
-
-          fileLoaderMatch.loader.exclude.push(/theme.config$/)
-          fileLoaderMatch.loader.exclude.push(/\.variables$/)
-          fileLoaderMatch.loader.exclude.push(/\.overrides$/)
-
-          return webpackConfig
-        },
-      },
-    },
-  ],
+  plugins: [{ plugin: require('@semantic-ui-react/craco-less') }],
 }
 ```
 

